mix.exs

```diff
@@ -19,7 +19,7 @@ defmodule RabbitMQ.MessageDeduplicationPlugin.Mixfile do
       extra_applications: [:rabbit],
       mod: {RabbitMQMessageDeduplication, []},
       registered: [RabbitMQMessageDeduplication],
-      broker_version_requirements: ["3.13.0"]
+      broker_version_requirements: ["3.13.0-3.13.x", "4.0.0-4.0.x"]
```

Comment:
Not sure about this syntax or whether that's an adequate config for the project. I just set the value that allowed me to enable the plugin on my manual tests (enabling the plugin).